docs/build.sbt

```diff
@@ -1,10 +1,10 @@
-val ScalaVersion = "2.11.12"
+val ScalaVersion = "2.12.4"
 
 val AkkaVersion = "2.5.8"
 val JUnitVersion = "4.11"
 val JUnitInterfaceVersion = "0.11"
 val ScalaTestVersion = "3.0.4"
-val PlayVersion = "2.6.9"
+val PlayVersion = "2.6.10"
```

Comment:
This will conflict with #1182. Was it needed for this change, or just something you were fixing up because you noticed it was out of sync?

Reply:
just bumped the version, not really needed. We can remove to avoid the conflict with #1182